src/containers/CheckoutPage.less

```diff
@@ -0,0 +1,364 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+ @import './../style/common.less';
+ 
+.checkout-container {
+  position: relative;
+
+  .checkout-container-inner {
+    margin-left: -15px;
+    margin-right: -15px;
+
+    .view-title {
+      color: #000000;
+      text-transform: capitalize;
+
+      @media screen and (max-width: @mobileWidth - 1px) {
+        text-align: center;
+      }
+    }
+
+    .checkout-new-address-btn,
+    .checkout-edit-address-btn,
+    .checkout-delete-address-btn,
+    .checkout-new-payment-btn
+    {
+      border-radius: 4;
+    }
+
+    .checkout-title-container {
+      h1 {
+        position: relative;
+        float: left;
+        width: 100%;
+        min-height: 1px;
+        padding-left: 15px;
+        padding-right: 15px;
+        margin-top: 64px;
+        margin-bottom: 64px;
+      }
+    }
+
+    @media screen and (max-width: @mobileWidth - 1px) {
+      font-size: 16px;
+    }
+
+    .checkout-sidebar {
+      float: left;
+      width: 100%;
+      position: relative;
+      min-height: 1px;
+      padding-left: 15px;
+      padding-right: 15px;
+
+      @media (min-width: 992px) {
+        float: left;
+        width: 33.33333333333333%;
+        text-align: right;
+      }
+
+      @media (min-width: 1200px) {
+        float: left;
+        width: 25%;
+      }
+
+      .checkout-sidebar-inner {
+        display: inline-block;
+        padding: 32px;
+        border: 3px solid #040060;
+        border-radius: 6px;
+        background-color: rgba(255, 255, 255, 0.6);
+        text-align: left;
+        position: relative;
+
+        .btn-cmd-submit-order {
+          display: inline-block;
+          padding: 6px 12px;
+          margin-bottom: 0;
+          font-size: 14px;
+          font-weight: normal;
```

Comment:
This is default, unless some parent sets it to bold, you can probably remove it.